packages/features/bookings/lib/getBookingResponsesSchema.ts

```diff
@@ -236,7 +236,7 @@ function preprocess<T extends z.ZodType>({
         }
 
         if (bookingField.type === "phone") {
-          if (!(await phoneSchema.safeParseAsync(value)).success) {
+          if (!bookingField.hidden && !(await phoneSchema.safeParseAsync(value)).success) {
```

Comment:
shouldn't we only check this when the bookingField is required?  

`if (bookingField.required && !(await phoneSchema.safeParseAsync(value)).success)`

Reply:
I don't think so. If the field is optional, then it doesn't perform any validity check on the number.
For eg:-
![Screenshot 2024-11-11 141246](https://github.com/user-attachments/assets/bce54ec6-10cd-48e8-a9b3-04f2b9c23b92)